### ISSUE TITLE
Annotation length is sometimes longer than the video length

### ISSUE THREAD

**Issue by kumatheworld** [wangxiao5791509/TNL2K_evaluation_toolkit#2]
In the following 12 videos out of the 1,300 training videos, the annotation length is longer than the video length.
| Video                     | #Frames | #BBoxes |
| :------------------------ | ------: | ------: |
| Assassin_video_5-Done     |     785 |    1080 |
| BF5_FireGUN_video_01_done |     633 |    1266 |
| Baseball_game_002-Done    |     713 |     714 |
| Boat_video1-Done          |     470 |     570 |
| Cat_BBC_video_03-Done     |     133 |     145 |
| Fight_video_02-Done       |     914 |     916 |
| Fight_video_04-Done       |     816 |     817 |
| Joker_video_X02_done      |    1268 |    2139 |
| catNight_video_B04_done   |     151 |    1518 |
| maliao_4-Done             |    1608 |    1609 |
| monitor_bikeyellow        |     929 |     930 |
| videoPlayer_video_01_done |    2008 |    2009 |

For those videos, which part of the annotations should we be using?

**Comment by wangxiao5791509** [comment]
@kumatheworld  Hi, thanks for your kind reminder. We will update the new annotations for these video sequences.

**Comment by wangxiao5791509** [comment]
@kumatheworld   Hi, I checked all the 12 videos and find that the annotation is insistently caused by the following reasons: 
1. the number of gt is larger than video frames by 1 or 2 frames; 
2. the inaccurate division of original long videos, for example, we divide long videos into several short-term videos; 
3. the late annotations of the video 'Fight_video_02-Done' are wrongly saved (maybe the bug of annotation toolkit). 

I have modified all these videos and the new annotation can be found at: [anno](https://github.com/wangxiao5791509/TNL2K_evaluation_toolkit/tree/main/annos/revised_annotations.zip)


Also, I will update these annotations in the BaiduYun, Onedrive, and GoogleDrive. Thanks for your kind reminder.
